.github/workflows/build_and_test.yml

```diff
@@ -126,144 +138,85 @@ jobs:
-#          path: ${{ env.BIN_DIR }}
-#
-#      - name: Download Client
-#        uses: dawidd6/action-download-artifact@v2
-#        with:
-#          github_token: ${{secrets.GITHUB_TOKEN}}
-#          workflow: build_and_test.yml
-#          run_id: 1033249727
-#          name: integritee-client-da8d4b442d3f2b09dbafb097d4d7a1bce409d518
-#          path: ${{ env.BIN_DIR }}
-#
-#      - name: Download Enclave
-#        uses: dawidd6/action-download-artifact@v2
-#        with:
-#          github_token: ${{secrets.GITHUB_TOKEN}}
-#          workflow: build_and_test.yml
-#          run_id: 1033249727
-#          name: enclave-signed-da8d4b442d3f2b09dbafb097d4d7a1bce409d518
-#          path: ${{ env.BIN_DIR }}
-
-      - name: Download integritee-node
-        uses: dawidd6/action-download-artifact@v2
-        with:
-          github_token: ${{secrets.GITHUB_TOKEN}}
-          workflow: ci.yml
-          name: integritee-node-dev-a1a80ab709c3c94f3174c7218f86b4de390d6dc5
-          # in fact this action should download the latest artifact, but sometimes fails. Then we need to
-          # set the `run_id` to force a download of an updated binary.
-          run_id: 2740081205
-          path: node
-          repo: integritee-network/integritee-node
-
-      - name: Prepare working directory
-        run: |
-          mkdir -p ${{ env.LOG_DIR}}
-          chmod +x node/integritee-node
-          cd ${{ env.BIN_DIR }}
-          chmod +x ${{ env.WORKER_BIN }}
-          chmod +x ${{ env.CLIENT_BIN }}
-          chmod +x ${{ env.ENCLAVE_BIN }}
-
-      - name: "Setup Keys"
+      - name: Load Worker & Client Images
         env:
-          KEY: ${{ secrets.IAS_PRIMARY_KEY }}
-          SPID: ${{ secrets.IAS_SPID }}
-          TLS_CERTIFICATE: ${{ secrets.TLS_WS_SERVER_CERTIFICATE }}
-          TLS_PRIVATE_KEY: ${{ secrets.TLS_WS_SERVER_PRIVATE_KEY }}
+          DOCKER_BUILDKIT: 1
+#        run: |
+#          cat integritee-worker-${{ matrix.mode }}-${{ github.sha }}.tar.gz | docker import - integritee-worker-${{ matrix.mode }}-${{ github.sha }}
+#          cat integritee-cli-client-${{ matrix.mode }}-${{ github.sha }}.tar.gz | docker import - integritee-cli-client-${{ matrix.mode }}-${{ github.sha }}
+#          docker images --all
         run: |
-          cd ${{ env.BIN_DIR }}
-          echo "${{ env.KEY }}" > key.txt
-          echo "${{ env.SPID }}" > spid.txt
-          echo "${{ env.TLS_CERTIFICATE }}" > end.fullchain
-          echo "${{ env.TLS_PRIVATE_KEY }}" > end.rsa
-          chmod 644 end.fullchain
-          chmod 644 end.rsa
-
-      - name: Run local setup
-        # * `set -eo pipefail` is needed to return an error even if piped to `tee`.
-        shell: bash --noprofile --norc -eo pipefail {0}
+          docker image load --input integritee-worker-${{ matrix.mode }}-${{ github.sha }}.tar.gz
+          docker image load --input integritee-cli-client-${{ matrix.mode }}-${{ github.sha }}.tar.gz
+          docker images --all
```

Comment:
Load the images and re-name the tags to match the ones in the docker-compose files